### PR TITLE
Fix CPHL names closes #532

### LINKED ISSUE #532: CPHL, CHLF and CHLU are the same parameter with different attributes
CPHL, CHLF and CHLU should be made distinct in the comment variable attribute:

CPHL: conversion from fluorescence to chlorophyll-a using uknown coefficients (most likely factory ones though, would be good to confirm).
CHLF: conversion from fluorescence to chlorophyll-a using factory coefficients.
CHLU: conversion from fluorescence to chlorophyll-a using user coefficients.

### PR DESCRIPTION
**Pull request by ocehugo**
After #594, #581, #590 this include:

1. Sandboxed the fixes.
2. Add some files that were missing for running the tests.

Renames of functions raised in #594 are in the next PR.

### CODE REVIEW

File: Parser/convertECOrawVar.m [hunk 1]
```diff
@@ -60,9 +60,7 @@
         
     case 'CHL' %ug/l (470/695nm)
         name = 'CPHL';
-        comment = ['Artificial chlorophyll data computed from bio-optical ' ...
-            'sensor raw counts measurements. Originally expressed in ' ...
-            'ug/l, 1l = 0.001m3 was assumed.'];
+        comment = getCPHLcomment('unknown','470nm','695nm');
```
Comment (lbesnard) [inline Parser/convertECOrawVar.m]: I think in the future, it would be good not to create new branches, PR's ... when working on a similar PR. It makes it really hard to follow with everyone's comments.
For example, why is this line changed? In your previous PR, it was 
```comment = getCPHLcomment('unknown');```
https://github.com/aodn/imos-toolbox/pull/581/files#diff-32be37807d5ec8e56a7cb8fc96406b52R63

Comment (ocehugo) [reply Parser/convertECOrawVar.m]: The function to create comments need to handle other custom entires (say excitation wavelength). Rebase was the only way. GG asked me to change commit order to make sure the code is not broken. The request of linter also required a rebase.

**Comment by ocehugo** [comment]
I just realize that one commit is missing - I forgot to cherry pick. Will add later - it's just a simple removal

File: Parser/convertECOrawVar.m (hunk 1 above)
Comment (lbesnard) [reply Parser/convertECOrawVar.m]: sorry, I don't understand what you're talking about. I'm saying that this line is 
``` comment = getCPHLcomment('unknown','470nm','695nm'); ```

while in the previous PR, it was 
``` comment = getCPHLcomment('unknown');```

Comment (ggalibert) [reply Parser/convertECOrawVar.m]: Yes, this is because Hugo did a bit more work that required ;-) and revisited the instrument documentation so that he can now specify which wavelengths are being used.

File: Parser/readWQMdat.m [hunk 2]
```diff
@@ -53,9 +53,9 @@
 %                   Dissolved Oxygen  ('DOXY'): mg/l
 %                   Dissolved Oxygen  ('DOX1'): mmol/m^3
 %                   Dissolved Oxygen  ('DOX'): ml/l
-%                   Chlorophyll       ('CPHL'): mg/m^3
-%                   Chlorophyll       ('CHLU'): mg/m^3   (user coefficient)
-%                   Chlorophyll       ('CHLF'): mg/m^3   (factory coefficient)
+%                   Chlorophyll       ('CPHL','CPHL_1','CPHL_2'): mg/m^3
+%                   Chlorophyll       ('CPHL','CPHL_1','CPHL_2'): mg/m^3   (user coefficient)
+%                   Chlorophyll       ('CPHL','CPHL_1','CPHL_2'): mg/m^3   (factory coefficient)
```
Comment (ggalibert) [inline Parser/readWQMdat.m]: Just checking: if there is only one chlorophyl var in the file then CPHL, otherwise CPHL_1, CPHL_2, CPHL_3, etc. See https://github.com/aodn/imos-toolbox/search?p=1&q=_2&unscoped_q=_2

File: Parser/readWQMraw.m [hunk 3]
```diff
@@ -595,18 +595,8 @@
...
-  'that fluoresces in the region of 695nm. '...
-  'Originally expressed in ug/l, 1l = 0.001m3 was assumed.'}};
-  params{end+1} = {'U_Cal_CHL',       {'CHLU', 'Artificial chlorophyll data '...
-  'computed from bio-optical sensor raw counts measurements using user calibration coefficient. The '...
-  'fluorometre is equipped with a 470nm peak wavelength LED to irradiate and a '...
-  'photodetector paired with an optical filter which measures everything '...
-  'that fluoresces in the region of 695nm. '...
-  'Originally expressed in ug/l, 1l = 0.001m3 was assumed.'}};
+  params{end + 1} = {'F_Cal_CHL', {'CHPL', getCPHLcomment('factory', '470nm', '695nm')}};
+  params{end + 1} = {'U_Cal_CHL', {'CHPL', getCPHLcomment('user', '470nm', '685nm')}};
```
Comment (ggalibert) [inline Parser/readWQMraw.m]: Typo CHPL still here in the above 2 lines.

File: Parser/convertSBEcnvVar.m [hunk 4]
```diff
@@ -121,13 +121,16 @@
...
     case 'flC'
       name = 'CPHL';
-      comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements. Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
+      comment = getCPHLcomment('unknown','430nm','685nm');
 
     %fluorescence (ug/l)
     %Aquatracka SBE19plus cdom sensor as cphl implemented on #550
     case 'flCUVA'
+
+      warning('Assuming `flCUVA` as from Aquatracka III instead of Aquatracka UV');
```
Comment (ggalibert) [inline Parser/convertSBEcnvVar.m]: I tend not to use warning() for this kind of message. Instead something like:
```Matlab
fprintf('%s\n', ['Warning : ' 'Assuming blablablah']);
```

File: Parser/convertECOrawVar.m (hunk 1 above)
Comment (ocehugo) [reply Parser/convertECOrawVar.m]: @lbesnard,

In the first round of the implementation, I look at all the comments. Some parsers had wavelength information, some didn't. So I took the decision to remove it for consistency. In the first review, GG said that wavelength info needs to be kept. Since this parser didn't have any prev. info about the wavelengths, I had to search for it (and since I was doing it, I also double-check all other wavelength comments). 

Finally, the function was extended with new arguments to build the comment correctly and explicitly.

File: Parser/convertSBEcnvVar.m (hunk 4 above)
Comment (ocehugo) [reply Parser/convertSBEcnvVar.m]: @gg, this makes no sense for me (and it's an anti-pattern). Any deeper reason? 

The warning message goes to the same location as the fprintf call (stdout) anyway. Is this just because the user can "warning('off')"?

File: Parser/readWQMraw.m (hunk 3 above)
Comment (ocehugo) [reply Parser/readWQMraw.m]: Done

File: Parser/readWQMdat.m (hunk 2 above)
Comment (ocehugo) [reply Parser/readWQMdat.m]: The comment is wrongly edited -  the rule is CPHL. If more, then the second is CPHL_2, CPHL_3 as is done in the toolbox for TEMP for example. 

Fixed in both raw and dat readers

**Comment by ocehugo** [comment]
@lbesnard , @ggalibert  - all points resolved - check changes and the last two commits.

File: Parser/convertSBEcnvVar.m (hunk 4 above)
Comment (ggalibert) [reply Parser/convertSBEcnvVar.m]: I cannot remember right now. Mostly is a comment for consistency with what's already in the code elsewhere.

File: Parser/readWQMdat.m (hunk 2 above)
Comment (ggalibert) [reply Parser/readWQMdat.m]: I just want to make sure with you. The rule should be:
-if only one chlorophyl var in the file, then CPHL.
-if more than 1 chlorophyl var in the file, then CPHL_1, CPHL_2, etc...
